src/request/derive-address/index.html

```diff
@@ -79,12 +79,12 @@
     <div id="app">
         <div id="unlock" class="page nq-card">
             <div class="page-header nq-card-header">
-                <h1 data-i18n="derive-address-heading-password" class="nq-h1">Unlock your account</h1>
+                <h1 data-i18n="derive-address-heading-password" class="nq-h1">Unlock account</h1>
```

Comment:
I think `account` should be capitalized here for consistency:
```suggestion
                <h1 data-i18n="derive-address-heading-password" class="nq-h1">Unlock Account</h1>
```

Reply:
missed that one.